<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditWindow</class>
 <widget class="CreditWindow" name="CreditWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Кредитный калькулятор</string>
  </property>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>160</width>
     <height>40</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>закрыть</string>
   </property>
  </widget>
  <widget class="QLabel" name="credit_duration">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Срок</string>
   </property>
  </widget>
  <widget class="QLabel" name="interest_rate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Процентная ставка</string>
   </property>
  </widget>
  <widget class="QLabel" name="credit_sum">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Сумма кредита</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_calculate">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>410</y>
     <width>160</width>
     <height>40</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_sum">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_months">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Месяцев</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Лет</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit_percent">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>360</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Метод расчета</string>
   </property>
   <widget class="QRadioButton" name="radioButton_differentiate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>170</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Дифференцированный</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_annuity">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Аннуитетный</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="monthly_payment_input">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Ежемесячный платёж</string>
   </property>
  </widget>
  <widget class="QLabel" name="overpayment_input">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Переплата</string>
   </property>
  </widget>
  <widget class="QLabel" name="sum_to_pay_input">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Итоговая сумма</string>
   </property>
  </widget>
  <widget class="QLabel" name="monthly_payment_output">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string></string>
   </property>
  </widget>
  <widget class="QLabel" name="overpayment_output">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>290</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string></string>
   </property>
  </widget>
  <widget class="QLabel" name="sum_to_pay_output">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>330</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string></string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CreditWindow</class>
   <extends>QWidget</extends>
   <header>creditwindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
